contrib/generate_profile.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+if [ $# -ne 1 ]
+  then
+    echo "Usage: generate_profile.sh Path_Of_Foundation_Build_Directory"
+    exit 1
+fi
+fdbdir=$1
+export LD_LIBRARY_PATH=$fdbdir/lib:$LD_LIBRARY_PATH
+export FDB_CLUSTER_FILE=$fdbdir/fdb.cluster
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/fdb-%p.profraw
+$fdbdir/bin/fdbmonitor --conffile $fdbdir/sandbox/foundationdb.conf --lockfile $fdbdir/sandbox/fdbmonitor.pid &
+# This profile will be ignored
+export LLVM_PROFILE_FILE=$fdbdir/sandbox/cli-%m.profraw
+$fdbdir/bin/fdbcli -C $fdbdir/fdb.cluster --exec 'configure new ssd single'
```

Comment:
Sorry, one more thing... should we consider making the storage engine as an argument?
Also, I'm not sure how effective/harmful this is, but should we consider supporting non-single replications?

Reply:
I think we can make the whole 'new ssd single' as an argument, named db_configure ?
Then the code become

$fdbdir/bin/fdbcli -C $fdbdir/fdb.cluster --exec 'configure '$db_configure

Reply:
maybe $fdbdir/bin/fdbcli -C $fdbdir/fdb.cluster --exec 'configure new '$db_configure

just put 'ssd single' to db_configure

Reply:
done